tasks.py

```diff
@@ -341,6 +341,20 @@ def post_upgrade(context):
     run_command(context, command)
 
 
+@task()
+def dumpdata(context, format="json"):
+    """Dump data from database to db_output file."""
+    command = f"nautobot-server dumpdata --exclude extras.job --indent 4 --output db_output.{format} --format {format}"
+    run_command(context, command)
+
+
+@task()
```

Comment:
Can you please add `help` for `--file-name` in keeping with other task functions? See how it's done with `makemigrations` above. ☝️